lib/orion.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const Intercom = require('intercom-client')
+const envIs = require('101/env-is')
+const exists = require('101/exists')
+const Company = require('./company')
+
+/**
+ * Orion, the huntsman, in charge of helping us hunt down those big customers.
+ * Used for communication with intercom with very little mucking about configuration
+ * @class
+ * @author Ryan Kahn
+ */
+class Orion {
+  /**
+   * Creates a new Orion instance for reporting to intercom
+   */
+  constructor () {
+    this.company = new Company(this)
```

Comment:
ahh I see, would it be better to name this class `EnvConfig` or something similar since this class's job is to use env to set `canUseIntercom`. non blocking.